spec/support/acts_as_matcher.rb

```diff
@@ -1,6 +1,6 @@
 require 'rspec/expectations'
 
-RSpec::Matchers.define :act_as do |actable|
+RSpec::Matchers.define :acts_as do |actable|
```

Comment:
Double check spelling of `:acts_as`

If `:acts_as` is the correct one, fix this in a separate PR with all the other corrections of `:act_as` to `:acts_as`.